tests/unit/py_proxy/views_test.py

```diff
@@ -26,5 +29,115 @@ class TestIncludeMe:
     assert config.add_route.call_args_list == [
         mock.call("index", "/"),
         mock.call("status", "/_status"),
+        mock.call("favicon", "/favicon.ico"),
+        mock.call("robots", "/robots.txt"),
+        mock.call("pdf", "/pdf/{pdf_url:.*}"),
+        mock.call("content_type", "/{url:.*}"),
     ]
     config.scan.assert_called_once_with("py_proxy.views")
+
+
+class TestFaviconRoute:
+    def test_returns_favicon_icon(self, pyramid_request):
+        result = views.favicon(pyramid_request)
+
+        assert result.content_type == "image/x-icon"
+        assert result.status_int == 200
+
+
+class TestRobotsTextRoute:
+    def test_returns_robots_test_file(self, pyramid_request):
+        result = views.robots(pyramid_request)
+
+        assert result.content_type == "text/plain"
+        assert result.status_int == 200
+
+
+class TestPdfRoute:
+    # def test_pdf_renders_pdf_template(self, pyramid_request):
+    #    pyramid_request.override_renderer = mock.PropertyMock()
+    #    result = views.pdf(pyramid_request)
+    #    import pdb; pdb.set_trace()
+
+    #    assert "pdfjs_viewer.html" in pyramid_request.override_renderer
```

Comment:
We shouldn't commit any commented-out code

Reply:
This is my placeholder for the jinja template rendering test. I was hoping to get it committed last night but it didn't happen-I have it locally now though. I'll commit it shortly.